client/src/components/ErrorMessage/ErrorMessage.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import styles from "./ErrorMessage.module.css";
+import { PropTypes } from "prop-types";
+
+export const ErrorMessage = ({ errorMessage }) => {
+  return errorMessage === "" ? (
+    <></>
```

Comment:
We can actually omit this check because this component will only be used upon an error

Reply:
The check to show an error message should be done on the actual form component (that contains the password/email since this can trigger the error)